java/server/src/org/openqa/selenium/remote/server/SessionCleaner.java

```diff
@@ -115,7 +116,8 @@ void checkExpiry() {
           try {
             deleteSession.call();
           } catch (Exception e) {
-            throw new RuntimeException(e);
+            log.log(Level.WARNING, "Could not delete session " + session.getSessionId(), e);
+            continue;
```

Comment:
In theory, that sounds like it might be a good idea, but in practice the exception is caused by the session already being closed and deleted. I have this running in production on a long running selenium server and I have never seen the log fire twice on the same session, meaning by the time the loop comes back around, the session has been cleaned out by some other part of the code.  I would rather repeatedly try to delete the session than assume that things are ok and remove this from the active sessions list. It is of utmost importance that we not leak browsers.